### PR TITLE
add Sentry config info to Rails deployment section

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

**Comment by michaelwehrley** [comment]
:+1: Looks great to me!

File: deployment/rails/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+if Rails.env.staging? || Rails.env.production?
+  require "raven"
+
+  Raven.configure do |config|
+    config.dsn = Rails.application.config.sentry_dsn
+  end
+end
+```
+
+On staging, add the following to the `config/environments/staging.rb` file
```
Comment (michaelwehrley) [inline deployment/rails/README.md:33]: @ericcf After going through this, do we need to include a section about setting up in production?

File: deployment/rails/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+within the `Application.configure do` block
```
Comment (michaelwehrley) [inline deployment/rails/README.md:34]: Is this analogous to `Rails.application.configure` in rails applications?

File: deployment/rails/README.md (hunk 1 above)
Comment (michaelwehrley) [inline deployment/rails/README.md:33]: Should we also note to add this to `.gitignore`?

Comment (ericcf) [inline deployment/rails/README.md:33]: We could add it for production, although it's analogous. Maybe I can update the instructions so they aren't redundant.

What are you saying should be added to `.gitignore`?

File: deployment/rails/README.md (hunk 2 above)
Comment (ericcf) [inline deployment/rails/README.md:34]: Yes, I guess it should be updated with the most recent conventions.

File: deployment/rails/README.md (hunk 1 above)
Comment (michaelwehrley) [inline deployment/rails/README.md:33]: If we don't, aren't we exposing our sentry_dsn token/API Keys?  Should we place them in an ENV variable on the server?

Comment (ericcf) [inline deployment/rails/README.md:33]: The `staging.rb` etc. files are only on the servers. They get symlinked when we deploy.

Comment (michaelwehrley) [inline deployment/rails/README.md:33]: Therefore, should we put the API Key in an ENV variable (i.e., `ENV["sentry"]`) and reference that in the ``staging.rb`?

Comment (ericcf) [inline deployment/rails/README.md:33]: The problem is that the keys are different for each project, so it wouldn't save us anything.

**Pull request merged**